.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/app.log
```

```suggestion
logs/
```
it is better not to add the whole folder, and not just one file